### ISSUE TITLE
Accessibility: The date picker on public pages is unusable for keyboard-only users

### ISSUE THREAD

**Issue by KreerC** [SergeyMosin/Appointments#214]
The date picker on public appointment booking pages is currently inaccessible to keyboard-navigating- and screen reader users. This should be fixed immediately to allow the tool to be used in crucial situations (see e.g. #212) as my employer currently plans to do.

Thank you for your hard work on this!

**Comment by SergeyMosin** [comment]
Note: Additionally compatibility with screen-reader software such as JAWS for Windows (Blind users) is needed.

**Comment by nkreer** [comment]
Before creating this issue, I've already verified that it's at least somewhat usable with the iPadOS 14 VoiceOver screen reader. Bugs with that should probably be their own issues.

**Comment by SergeyMosin** [comment]
fixed in v1.14.6
